scan/tls_handshake.go

```diff
@@ -71,11 +124,15 @@ type cipherVersionList []cipherVersions
 func (cvList cipherVersionList) String() string {
 	cvStrings := make([]string, len(cvList))
 	for i, c := range cvList {
-		versStrings := make([]string, len(c.versions))
-		for j, vers := range c.versions {
-			versStrings[j] = tls.Versions[vers]
+		versStrings := make([]string, len(c.data))
+		for j, d := range c.data {
+			curveStrings := make([]string, len(d.curves))
+			for k, c := range d.curves {
+				curveStrings[k] = fmt.Sprintf("%d", tls.Curves[c])
```

Comment:
This should just be `curveStrings[k] = tls.Curves[c]`, as it's already a string.